src/saving/SaveInformation.cs

```diff
@@ -51,6 +57,9 @@ public enum SaveType
 
     public SaveType Type { get; set; } = SaveType.Manual;
 
+    [JsonIgnore]
+    public string TypeString => Type.GetAttribute<DescriptionAttribute>().Description;
```

Comment:
I think this should be named something like "TranslatedSaveTypeString", and do the translate call, so SaveListItem doesn't need to do it (as I foresee a little bit of a chance that other code will also want to show the save type).
`public string TranslatedSaveTypeString => TranslationServer.Translate(Type.GetAttribute<DescriptionAttribute>().Description);`